style.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  background-color: #1c1a19;
+  background-image: url("Group.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  padding-top: 120px;
+  padding-bottom: 120px;
+  width: 375px;
+  height: 590px;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: #3c3a3c;
+}
+
+.bars {
+  color: #dbd8d7;
+  padding-right: 30px;
+}
+
+img {
+  color: #fff;
+}
+
+.intro {
+  color: #ff6b00;
+  font-size: 56px;
+  font-family: 'Crete Round', serif;
+  margin: 10px 24px 10px 24px;
+}
+
+h3 {
+  color: #fff5e1;
+  font-size: 16px;
+}
+
+h2 {
+  color: #fff5e1;
+  font-size: 26px;
+  font-family: sans-serif;
+  margin: 10px 24px 10px 24px;
+}
+
+.logo {
+  font-size: 16px;
+  text-decoration: none;
+  color: #dbd8d7;
+  padding-left: 30px;
+}
+
+p {
+  color: #fff;
+  font-size: 20px;
+  font-family: 'Times New Roman', Times, serif;
+  margin: 10px 24px 10px 24px;
+}
+
+a {
+  color: #dbd8d7;
+}
```

Comment:
I think it's better to use classes than HTML tags. CSS Classes will help you stylize HTML elements quickly.